File: py/problem005.py
"""
2520 is the smallest number that can be divided 
by each of the numbers from 1 to 10 without 
any remainder.

What is the smallest positive number that is evenly 
divisible by all of the numbers from 1 to 20?
"""
from Helper import lcm

lnumber = 1
for i in range(1, 21):
	lnumber = lcm(lnumber, i)

print lnumber	

File: c/problem005.c
/**
*2520 is the smallest number that can be divided 
*by each of the numbers from 1 to 10 without 
*any remainder.
*
*What is the smallest positive number that is evenly 
*divisible by all of the numbers from 1 to 20?
*/
#include <stdio.h>

void main() 
{
	long lnum = 1;
	int i;
	for (i = 1; i <= 20; i++) {
		lnum = lcm(lnum, i);
	}
	printf("%ld\n", lnum);
}